HeterogeneousCore/CUDACore/test/test_CUDAScopedContextKernels.cu

```diff
@@ -4,21 +4,13 @@
 #include <cuda_runtime.h>
 
 namespace {
-  __global__
-  void single_mul(int *d) {
-    d[0] = d[0]*2;
-  }
+  __global__ void single_mul(int *d) { d[0] = d[0] * 2; }
```

Comment:
I would really like to keep the `__global__` on a separate line... do we know if there is a settings for the function attributed in clang-format ?

Reply:
I don't know. I'd suggest to open an issue to `cmssw` repo to discuss.